### ISSUE TITLE
Build menuconfig script fails

### ISSUE THREAD

**Issue by jimwims** [loboris/MicroPython_ESP32_psRAM_LoBo#64]
I am trying to update to the latest version following the instructions now included in the wiki.  I have done this previously with no problems.  This time after I run ' git pull',  when I try to run './BUILD.sh menuconfig', I get the following:

      MENUCONFIG
      'make menuconfig' FAILED!

Where have I gone wrong here?

Jim

**Comment by jimwims** [comment]
I am using macOS 10.13.3 with an ESP-WROVER-KIT v3.

Jim

**Comment by loboris** [comment]
I have tried it on my MacOS High Sierra in VirtualBox (MicroPython not updated for more than a month).
`git pull` -> `./BUILD.sh menuconfig` -> `./BUILD.sh -j8 clean all` and there were no errors.

You can try to delete `MicroPython_ESP32_psRAM_LoBo/Tools/ver20xxxxxx.id` and then run `./BUILD.sh menuconfig`, the esp-idf and toolchain will be deleted and unpacked and menuconfig should run.

If nothing helps, try to make a fresh clone...
```
Boriss-iMac:MicroPython_ESP32_psRAM_LoBo boris$ git pull
remote: Counting objects: 775, done.
remote: Compressing objects: 100% (41/41), done.
remote: Total 775 (delta 452), reused 488 (delta 445), pack-reused 277
Receiving objects: 100% (775/775), 142.08 MiB | 1.66 MiB/s, done.
Resolving deltas: 100% (532/532), completed with 142 local objects.
From https://github.com/loboris/MicroPython_ESP32_psRAM_LoBo
   cc836a3..d775bb1  master     -> origin/master
Updating cc836a3..d775bb1
Fast-forward
 Documents/FT800CB_SCH.pdf                                                                      |  Bin 0 -> 47318 bytes
 Documents/FT81X_Series_Programmer_Guide.pdf                                                    |  Bin 0 -> 3605853 bytes
 Documents/FT81x_Data_Sheet.pdf                                                                 |  Bin 0 -> 2204450 bytes
 MicroPython_BUILD/.gitignore                                                                   |    5 +-
 MicroPython_BUILD/BUILD.sh                                                                     |   14 +-
 MicroPython_BUILD/build_func.sh                                                                |   47 ++-
 MicroPython_BUILD/components/curl/LICENSE-MIXING.md                                            |  127 +++++++
 MicroPython_BUILD/components/curl/README                                                       |   49 +++
 MicroPython_BUILD/components/curl/README_curl.md                                               |   51 +++
 MicroPython_BUILD/components/curl/esp32_curl_paches.zip                                        |  Bin 3483 -> 8512 bytes
 MicroPython_BUILD/components/curl/include/curl/system.h                                        |    7 +-
 MicroPython_BUILD/components/curl/lib/asyn-ares.c                                              |    2 -
 MicroPython_BUILD/components/curl/lib/config-dos.h                                             |    1 -
 MicroPython_BUILD/components/curl/lib/config-symbian.h                                         |    3 -
 MicroPython_BUILD/components/curl/lib/config-tpf.h                                             |    3 -
 MicroPython_BUILD/components/curl/lib/config-vxworks.h                                         |    3 -
 MicroPython_BUILD/components/curl/lib/config-win32.h                                           |    3 -
 MicroPython_BUILD/components/curl/lib/config-win32ce.h                                         |    3 -
 MicroPython_BUILD/components/curl/lib/curl_config.h                                            |   24 +-
 MicroPython_BUILD/components/curl/lib/curl_fnmatch.c                                           |   16 +-
 MicroPython_BUILD/components/curl/lib/curl_setup.h                                             |   10 +-
 MicroPython_BUILD/components/curl/lib/easy.c                                                   |   24 +-
 MicroPython_BUILD/components/curl/lib/http.c                                                   |   52 ++-
 MicroPython_BUILD/components/curl/lib/http2.c                                                  |    4 +-
 MicroPython_BUILD/components/curl/lib/imap.c                                                   |    8 +-
 MicroPython_BUILD/components/curl/lib/mime.c                                                   |   81 ++++-
 MicroPython_BUILD/components/curl/lib/mime.h                                                   |    3 +-
 MicroPython_BUILD/components/curl/lib/multi.c                                                  |    3 +-
 MicroPython_BUILD/components/curl/lib/parsedate.c                                              |    2 -
 MicroPython_BUILD/components/curl/lib/pop3.c                                                   |    8 +-
 MicroPython_BUILD/components/curl/lib/progress.c                                               |   20 +-
 MicroPython_BUILD/components/curl/lib/security.c                                               |    2 -
 MicroPython_BUILD/components/curl/lib/setopt.c                                                 |   55 ++-
 MicroPython_BUILD/components/curl/lib/setopt.h                                                 |    4 +-
 MicroPython_BUILD/components/curl/lib/smb.c                                                    |   29 +-
 MicroPython_BUILD/components/curl/lib/smb.h                                                    |   33 +-
 MicroPython_BUILD/components/curl/lib/smtp.c                                                   |    8 +-
 MicroPython_BUILD/components/curl/lib/ssh-libssh.c                                             |    2 -
 MicroPython_BUILD/components/curl/lib/ssh.c                                                    |   11 +-
 MicroPython_BUILD/components/curl/lib/strtoofft.h                                              |   10 +-
 MicroPython_BUILD/components/curl/lib/url.c                                                    |    2 -
 MicroPython_BUILD/components/curl/lib/url.h                                                    |    4 +-
 MicroPython_BUILD/components/curl/lib/urldata.h                                                |    4 +-
 MicroPython_BUILD/components/curl/lib/vtls/cyassl.c                                            |    2 -
 MicroPython_BUILD/components/curl/lib/vtls/darwinssl.c                                         |    2 -
 MicroPython_BUILD/components/curl/lib/vtls/gskit.c                                             |    4 +-
 MicroPython_BUILD/components/curl/lib/vtls/openssl.c                                           |   36 +-
 MicroPython_BUILD/components/libssh2/COPYING                                                   |   42 +++
 MicroPython_BUILD/components/libssh2/README.md                                                 |   16 +
 MicroPython_BUILD/components/libssh2/RELEASE-NOTES                                             |   31 ++
 MicroPython_BUILD/components/libssh2/src/libssh2_config.h                                      |    4 +
 MicroPython_BUILD/components/micropython/Kconfig.projbuild                                     |   40 +-
 MicroPython_BUILD/components/micropython/component.mk                                          |   49 ++-
 MicroPython_BUILD/components/micropython/esp32/help.c                                          |    9 +-
 MicroPython_BUILD/components/micropython/esp32/libs/espcurl.c                                  |  253 +++++++++----
 MicroPython_BUILD/components/micropython/esp32/libs/espcurl.h                                  |   71 +++-
 MicroPython_BUILD/components/micropython/esp32/libs/eve/FT8.h                                  |  736 +++++++++++++++++++++++++++++++++++++
 MicroPython_BUILD/components/micropython/esp32/libs/eve/FT8_commands.c                         | 1136 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 MicroPython_BUILD/components/micropython/esp32/libs/eve/FT8_commands.h                         |  322 ++++++++++++++++
 MicroPython_BUILD/components/micropython/esp32/libs/eve/LICENSE                                |   21 ++
 MicroPython_BUILD/components/micropython/esp32/libs/ftp.c                                      |  232 +++++++-----
 MicroPython_BUILD/components/micropython/esp32/libs/random.c                                   |   83 -----
 MicroPython_BUILD/components/micropython/esp32/libs/random.h                                   |   18 -
 MicroPython_BUILD/components/micropython/esp32/libs/tft/tftspi.c                               |   17 +
 MicroPython_BUILD/components/micropython/esp32/libs/tft/tftspi.h                               |    6 +-
 MicroPython_BUILD/components/micropython/esp32/machine_hw_spi.c                                |    6 +
 MicroPython_BUILD/components/micropython/esp32/machine_rtc.c                                   |    6 -
 MicroPython_BUILD/components/micropython/esp32/machine_timer.c                                 |   48 ++-
 MicroPython_BUILD/components/micropython/esp32/machine_uart.c                                  |  751 +++++++++++++++++++++++++++++++-------
 MicroPython_BUILD/components/micropython/esp32/main.c                                          |   18 +-
 MicroPython_BUILD/components/micropython/esp32/modcurl.c                                       |   49 ++-
 MicroPython_BUILD/components/micropython/esp32/moddisplay.c                                    |   31 +-
 MicroPython_BUILD/components/micropython/esp32/modgsm.c                                        |    2 +-
 MicroPython_BUILD/components/micropython/esp32/modmachine.c                                    |    9 +-
 MicroPython_BUILD/components/micropython/esp32/modmqtt.c                                       |   26 +-
 MicroPython_BUILD/components/micropython/esp32/modnetwork.c                                    |  384 +++++++++++++++-----
 MicroPython_BUILD/components/micropython/esp32/modnetwork.h                                    |    9 +-
 MicroPython_BUILD/components/micropython/esp32/modota.c                                        |   32 +-
 MicroPython_BUILD/components/micropython/esp32/modsocket.c                                     |   14 +
 MicroPython_BUILD/components/micropython/esp32/modssh.c                                        |  121 ++++--
 MicroPython_BUILD/components/micropython/esp32/modules/README.md                               |   19 +
 MicroPython_BUILD/components/micropython/esp32/modules/microWebSocket.py                       |  266 ++++++++++++++
 MicroPython_BUILD/components/micropython/esp32/modules/microWebSrv.py                          |  348 ++++++++++--------
 MicroPython_BUILD/components/micropython/esp32/modules/microWebSrv.py.orig                     |  706 -----------------------------------
 MicroPython_BUILD/components/micropython/esp32/modules/microWebTemplate.py                     |   25 +-
 MicroPython_BUILD/components/micropython/esp32/modules/upysh.py                                |    6 +-
 MicroPython_BUILD/components/micropython/esp32/modules_examples/bme280.py                      |   53 +++
 MicroPython_BUILD/components/micropython/esp32/modules_examples/lora/LICENSE                   |  674 ++++++++++++++++++++++++++++++++++
 MicroPython_BUILD/components/micropython/esp32/modules_examples/lora/README.md                 |    5 +
 MicroPython_BUILD/components/micropython/esp32/modules_examples/tft/tftdemo.py                 |   11 +-
 MicroPython_BUILD/components/micropython/esp32/{modules => modules_examples}/uftpserver.py     |   50 ++-
 MicroPython_BUILD/components/micropython/esp32/modules_examples/webserver/README.md            |    3 +
 MicroPython_BUILD/components/micropython/esp32/modules_examples/webserver/readme.txt           |    2 -
 MicroPython_BUILD/components/micropython/esp32/modules_examples/webserver/webserver_example.py |  105 ++++--
 MicroPython_BUILD/components/micropython/esp32/modules_examples/webserver/www/wstest.html      |   78 ++++
 MicroPython_BUILD/components/micropython/esp32/mpconfigport.h                                  |   13 +-
 MicroPython_BUILD/components/micropython/esp32/mphalport.c                                     |   13 +-
 MicroPython_BUILD/components/micropython/esp32/mpthreadport.c                                  |   58 ++-
 MicroPython_BUILD/components/micropython/esp32/mpthreadport.h                                  |    5 +-
 MicroPython_BUILD/components/micropython/esp32/mpversion.h                                     |   10 +-
 MicroPython_BUILD/components/micropython/esp32/network_mdns.c                                  |  290 ++++++++-------
 MicroPython_BUILD/components/micropython/extmod/modutimeq.c                                    |   18 +-
 MicroPython_BUILD/components/micropython/extmod/utime_mphal.c                                  |    8 +-
 MicroPython_BUILD/components/micropython/lib/mp-readline/readline.c                            |    2 +-
 MicroPython_BUILD/components/micropython/py/builtinhelp.c                                      |    4 +-
 MicroPython_BUILD/components/micropython/py/modthread.c                                        |   43 ++-
 MicroPython_BUILD/components/micropython/py/mpconfig.h                                         |    1 +
 MicroPython_BUILD/components/micropython/py/mphal.h                                            |    6 +-
 MicroPython_BUILD/components/micropython/py/mpstate.h                                          |    1 +
 MicroPython_BUILD/components/micropython/py/mpz.c                                              |   41 +++
 MicroPython_BUILD/components/micropython/py/mpz.h                                              |    2 +
 MicroPython_BUILD/components/micropython/py/obj.c                                              |   22 ++
 MicroPython_BUILD/components/micropython/py/obj.h                                              |    2 +
 MicroPython_BUILD/components/micropython/py/objint_mpz.c                                       |   15 +
 MicroPython_BUILD/components/micropython/py/runtime.c                                          |    1 +
 MicroPython_BUILD/components/micropython/py/vm.c                                               |    5 +-
 MicroPython_BUILD/components/quickmail/AUTHORS                                                 |    1 +
 MicroPython_BUILD/components/quickmail/COPYING                                                 |  674 ++++++++++++++++++++++++++++++++++
 MicroPython_BUILD/components/quickmail/ChangeLog                                               |  249 +++++++++++++
 MicroPython_BUILD/components/quickmail/README                                                  |   17 +
 MicroPython_BUILD/components/zlib/ChangeLog                                                    | 1515 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 MicroPython_BUILD/components/zlib/README                                                       |  115 ++++++
 MicroPython_BUILD/firmware/MicroPython_LoBo_Firmwares.zip                                      |  Bin 4432767 -> 0 bytes
 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32.zip                                          |  Bin 0 -> 814940 bytes
 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_all.zip                                      |  Bin 0 -> 1088938 bytes
 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_ota.zip                                      |  Bin 0 -> 822680 bytes
 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_psram.zip                                    |  Bin 0 -> 859835 bytes
 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_psram_all.zip                                |  Bin 0 -> 1140369 bytes
 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_psram_ota.zip                                |  Bin 0 -> 867547 bytes
 MicroPython_BUILD/firmware/README.md                                                           |    3 +-
 MicroPython_BUILD/firmware/esp32/MicroPython.bin                                               |  Bin 1302416 -> 1325984 bytes
 MicroPython_BUILD/firmware/esp32/bootloader/bootloader.bin                                     |  Bin 19568 -> 19600 bytes
 MicroPython_BUILD/firmware/esp32/partitions_mpy.bin                                            |  Bin 3072 -> 3072 bytes
 MicroPython_BUILD/firmware/esp32/partitions_mpy.csv                                            |    2 +-
 MicroPython_BUILD/firmware/esp32/sdkconfig                                                     |    2 +
 MicroPython_BUILD/firmware/esp32_all/MicroPython.bin                                           |  Bin 0 -> 1765168 bytes
 MicroPython_BUILD/firmware/esp32_all/bootloader/bootloader.bin                                 |  Bin 0 -> 19600 bytes
 MicroPython_BUILD/firmware/esp32_all/flash.sh                                                  |    2 +
 MicroPython_BUILD/firmware/esp32_all/partitions_mpy.bin                                        |  Bin 0 -> 3072 bytes
 MicroPython_BUILD/firmware/esp32_all/phy_init_data.bin                                         |  Bin 0 -> 144 bytes
 MicroPython_BUILD/{sdkconfig.fw_psram_all => firmware/esp32_all/sdkconfig}                     |   31 +-
 MicroPython_BUILD/firmware/esp32_ota/MicroPython.bin                                           |  Bin 1315808 -> 1339600 bytes
 MicroPython_BUILD/firmware/esp32_ota/bootloader/bootloader.bin                                 |  Bin 19568 -> 19600 bytes
 MicroPython_BUILD/firmware/esp32_ota/sdkconfig                                                 |    2 +
 MicroPython_BUILD/firmware/esp32_psram/MicroPython.bin                                         |  Bin 1435328 -> 1460864 bytes
 MicroPython_BUILD/firmware/esp32_psram/bootloader/bootloader.bin                               |  Bin 19568 -> 19600 bytes
 MicroPython_BUILD/firmware/esp32_psram/partitions_mpy.bin                                      |  Bin 3072 -> 3072 bytes
 MicroPython_BUILD/firmware/esp32_psram/partitions_mpy.csv                                      |    2 +-
 MicroPython_BUILD/firmware/esp32_psram/sdkconfig                                               |    2 +
 MicroPython_BUILD/firmware/esp32_psram_all/MicroPython.bin                                     |  Bin 1923120 -> 1936160 bytes
 MicroPython_BUILD/firmware/esp32_psram_all/bootloader/bootloader.bin                           |  Bin 19568 -> 19600 bytes
 MicroPython_BUILD/firmware/esp32_psram_all/sdkconfig                                           |    3 +
 MicroPython_BUILD/firmware/esp32_psram_ota/MicroPython.bin                                     |  Bin 1449216 -> 1474896 bytes
 MicroPython_BUILD/firmware/esp32_psram_ota/bootloader/bootloader.bin                           |  Bin 19568 -> 19600 bytes
 MicroPython_BUILD/firmware/esp32_psram_ota/sdkconfig                                           |    2 +
 MicroPython_BUILD/make_firmwares.sh                                                            |   82 -----
 MicroPython_BUILD/sdkconfig.defaults                                                           |   24 +-
 README.md                                                                                      |   44 ++-
 Tools/Linux/xtensa-esp32-elf_arm.tar.xz                                                        |  Bin 0 -> 24710780 bytes
 Tools/esp-idf.tar.xz                                                                           |  Bin 18648104 -> 18697048 bytes
 160 files changed, 8984 insertions(+), 2072 deletions(-)
 create mode 100644 Documents/FT800CB_SCH.pdf
 create mode 100644 Documents/FT81X_Series_Programmer_Guide.pdf
 create mode 100644 Documents/FT81x_Data_Sheet.pdf
 create mode 100644 MicroPython_BUILD/components/curl/LICENSE-MIXING.md
 create mode 100644 MicroPython_BUILD/components/curl/README
 create mode 100644 MicroPython_BUILD/components/curl/README_curl.md
 create mode 100644 MicroPython_BUILD/components/libssh2/COPYING
 create mode 100644 MicroPython_BUILD/components/libssh2/README.md
 create mode 100644 MicroPython_BUILD/components/libssh2/RELEASE-NOTES
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/libs/eve/FT8.h
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/libs/eve/FT8_commands.c
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/libs/eve/FT8_commands.h
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/libs/eve/LICENSE
 delete mode 100644 MicroPython_BUILD/components/micropython/esp32/libs/random.c
 delete mode 100644 MicroPython_BUILD/components/micropython/esp32/libs/random.h
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/modules/README.md
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/modules/microWebSocket.py
 delete mode 100644 MicroPython_BUILD/components/micropython/esp32/modules/microWebSrv.py.orig
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/modules_examples/lora/LICENSE
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/modules_examples/lora/README.md
 rename MicroPython_BUILD/components/micropython/esp32/{modules => modules_examples}/uftpserver.py (92%)
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/modules_examples/webserver/README.md
 delete mode 100644 MicroPython_BUILD/components/micropython/esp32/modules_examples/webserver/readme.txt
 create mode 100644 MicroPython_BUILD/components/micropython/esp32/modules_examples/webserver/www/wstest.html
 create mode 100644 MicroPython_BUILD/components/quickmail/AUTHORS
 create mode 100644 MicroPython_BUILD/components/quickmail/COPYING
 create mode 100644 MicroPython_BUILD/components/quickmail/ChangeLog
 create mode 100644 MicroPython_BUILD/components/quickmail/README
 create mode 100644 MicroPython_BUILD/components/zlib/ChangeLog
 create mode 100644 MicroPython_BUILD/components/zlib/README
 delete mode 100644 MicroPython_BUILD/firmware/MicroPython_LoBo_Firmwares.zip
 create mode 100644 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32.zip
 create mode 100644 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_all.zip
 create mode 100644 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_ota.zip
 create mode 100644 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_psram.zip
 create mode 100644 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_psram_all.zip
 create mode 100644 MicroPython_BUILD/firmware/MicroPython_LoBo_esp32_psram_ota.zip
 create mode 100644 MicroPython_BUILD/firmware/esp32_all/MicroPython.bin
 create mode 100644 MicroPython_BUILD/firmware/esp32_all/bootloader/bootloader.bin
 create mode 100755 MicroPython_BUILD/firmware/esp32_all/flash.sh
 create mode 100644 MicroPython_BUILD/firmware/esp32_all/partitions_mpy.bin
 create mode 100644 MicroPython_BUILD/firmware/esp32_all/phy_init_data.bin
 rename MicroPython_BUILD/{sdkconfig.fw_psram_all => firmware/esp32_all/sdkconfig} (95%)
 delete mode 100755 MicroPython_BUILD/make_firmwares.sh
 create mode 100644 Tools/Linux/xtensa-esp32-elf_arm.tar.xz
Boriss-iMac:MicroPython_ESP32_psRAM_LoBo boris$ cd MicroPython_BUILD
Boriss-iMac:MicroPython_BUILD boris$ ./BUILD.sh menuconfig
Removing old tools versions and cleaning build...
unpacking 'esp-idf'
unpacking MacOS/xtensa-esp32-elf

** Restored 'sdkconfig' **'

cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o mconf.o mconf.c
flex -L -P zconf -o zconf.lex.c zconf.l
bison -t -l -p zconf -o zconf.tab.c zconf.y
sed -E "s/\\x0D$//" zconf.gperf | gperf -t --output-file zconf.hash.c -a -C -E -g -k '1,3,$' -p -t
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o zconf.tab.o zconf.tab.c
lxdialog/check-lxdialog.sh -check cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD -lncurses 
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o lxdialog/checklist.o lxdialog/checklist.c
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o lxdialog/util.o lxdialog/util.c
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o lxdialog/inputbox.o lxdialog/inputbox.c
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o lxdialog/textbox.o lxdialog/textbox.c
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o lxdialog/yesno.o lxdialog/yesno.c
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o lxdialog/menubox.o lxdialog/menubox.c
cc -o mconf mconf.o zconf.tab.o lxdialog/checklist.o lxdialog/util.o lxdialog/inputbox.o lxdialog/textbox.o lxdialog/yesno.o lxdialog/menubox.o -lncurses 
cc  -DCURSES_LOC="<ncurses.h>" -DKBUILD_NO_NLS -Wno-format-security  -DLOCALE -MD   -c -o conf.o conf.c
cc -o conf conf.o  zconf.tab.o -lncurses 
MENUCONFIG
configuration written to /Users/boris/esp32/MicroPython_ESP32_psRAM_LoBo/MicroPython_BUILD/sdkconfig

*** End of the configuration.
*** Execute 'make' to start the build or try 'make help'.

GENCONFIG

---------------------
MicroPython for ESP32
---------------------

MENUCONFIG


*** End of the configuration.
*** Execute 'make' to start the build or try 'make help'.

OK.

Boriss-iMac:MicroPython_BUILD boris$ ./BUILD.sh -j8 clean all

---------------------
MicroPython for ESP32
---------------------

=============================
Cleaning MicroPython build...
=============================
OK.

==================
Building mpy-cross
OK.
==================
================================
Building MicroPython firmware...
================================
Restarting build ...
OK.
--------------------------------
Build complete.
You can now run ./BUILD.sh flash
to deploy the firmware to ESP32
--------------------------------

Boriss-iMac:MicroPython_BUILD boris$ 
```

**Comment by jimwims** [comment]
I was trying to do the build in a terminal window in Atom.  Through a rather circuitous process, I learned that I had not allowed for enough lines in the terminal window.  I went back to using the standard Apple 20 x 80 terminal and everything seems to be working fine.

Thanks for your help.

Is there way to make a donation to you other than through PayPal?  I  have had bad experiences with PP in the past so I avoid using it.
